e2e-tests/tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
```

Comment:
This is e2e-tests specific, so it should stay here for now.

If it's **super important** that we move files to the repository root, then we need to do an audit about all the systems that are looking at files in the root, what they do and make sure they're not interfering with one another when we make changes to those files. Alternatively, maybe this is the compelling reason we move the e2e-tests back into their own repository.